tutorials/physics/using_character_body_2d.rst

```diff
@@ -136,6 +136,13 @@ and ``get_slide_collision()``:
         var collision = get_slide_collision(i)
         print("I collided with ", collision.get_collider().name)
 
+    # if statement:
+    move_and_slide()
+    for i in get_slide_collision_count():
+        var collision = get_slide_collision(i)
+        if collision.get_collider().name == "TileMapLayer" || collision.get_collider().name == "RigidBody2D":
```

Comment:
```suggestion
        if collision.get_collider().name == "TileMapLayer" or collision.get_collider().name == "RigidBody2D":
```
Per the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html#boolean-operators)